### ISSUE TITLE
Add ability to not count Other players doors for own Door limit

### ISSUE THREAD

**Issue by Chris6ix** [PimvanderLoos/AnimatedArchitecture#225]
My players are a bit annoyed by the fact that when a friend gives them control over one of their doors with `/BigDoors AddOwner`, it counts as one of their own doors for `bigdoors.own.<numberOfDoors>`, meaning that they can't actually own the amount of Doors that are advertised and feel it is misleading. (e.g. 4 Doors using `bigdoors.own.4` are actually only 2 Doors if you also have permission to use 2 Doors of your Friend)
I thought a bit about it and i also agree with them.

So i wanted to ask you if you could either stop the doors of other players being counted for your own limit or at least add an option to enable that behavior (To not cause problems for servers that like the way it currently is.)

Thank you in advance :)

**Comment by PimvanderLoos** [comment]
Yeah, that sounds entirely reasonable.

I've added a new config option that will allow you to change this. It allows you to specify the level of ownership to include. This means that `0` will include only doors the player created, and `1` and `2` will also include admin and user-level shared ownerships. This works additively, so `2` includes all doors, just like it currently does.

I feel like only including created doors makes the most sense (although admin-level ownership is a close second), so I've made that the default.

Pending a full release (planned for tomorrow, provided nothing else comes up), I've published the change in [dev-build 1122](https://pim16aap2.nl/BigDoors/b1122/).
